examples/currency-transfer.yaml

```diff
@@ -0,0 +1,177 @@
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - bob: https://bob.example/
+    alice: https://alice.example/
+    abank: https://abank.example/
+    bbank: https://bbank.example/
+    notary: https://notary.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/currency-transfer.yaml
+'@graph':
+
+  # accounts before
+
+  - '@id': abank:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    '@type': EconomicResource
+    skos:note: shared account Alice has some credits on
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 200
+    share:
```

Comment:
what @type is share?

Reply:
I didn't want to make up name since it doesn't seem to matter how we would call something like that, it pretty much plays role of [Qualified Relation](http://patterns.dataincubator.org/book/qualified-relation.html) between resource and agent. Flows should include all information necessary to calculate it.

Reply:
OK, just wanted to know if it was an EconomicResource or not.